aepp/src/utils/aeternityNetwork.js

```diff
@@ -86,31 +85,11 @@ aeternity.hasActiveWallet = () => {
   return !!aeternity.client;
 };
 
-/**
- * Checks if the initialized client is connected to the ae-mainnet
- * @returns {boolean}
- */
-aeternity.isMainnet = () => {
-  return aeternity.networkId === 'ae_mainnet';
-};
-
-
 /**
  * Initializes the aeternity sdk to be imported in other occasions
  * @returns {Promise<boolean>}
  */
 aeternity.initClient = async () => {
-  if (process && process.env && process.env.PRIVATE_KEY && process.env.PUBLIC_KEY) {
```

Comment:
i think this is relevant for the tests

Reply:
are they still working at all, at least I couldn't see those envs defined anywhere.

Reply:
does not look like we ever wrote any :shrug: